### DISCUSSION TITLE
filter date picker not working in 1.11.4

### DISCUSSION THREAD

**Discussion by henryw374** [freeplane/freeplane#1960] [Help]
**Describe the bug**

unable to use any date filter 

**To Reproduce**

show filters
choose date filter from filter menu
choose e.g. 'modified before'
date picker appears
now unable to choose date on the date picker 
changing values in the picker does not change the date in the drop down.
double clicking on a date or on the drop down menu does not close the picker
the picker cannot be closed and is overlaying all other applications,

**Expected behavior**

should be able to pick date from picker, or type in a date

**Mind map files**
If possible, upload a zipped mind map file where the problem occurs.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: ubuntu 24.04
 - Version 1.11.4
 - Java Version 21

**Additional context**

**Comment by macmarrum** [comment]
On ArchLinux, I can select a date in the date picker, but the chosen value is applied in the filter only once I click away from the date picker so that it is closed.
